package.json

```diff
@@ -426,25 +429,24 @@
     "yauzl": "^2.10.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.16.0",
-    "@babel/core": "^7.16.0",
+    "@babel/cli": "^7.21.0",
+    "@babel/core": "^7.21.0",
     "@babel/eslint-parser": "^7.16.3",
     "@babel/eslint-plugin": "^7.14.5",
-    "@babel/generator": "^7.16.0",
```

Comment:
why have we removed @babel/generator into a prod dependency? I dont think it is one. In addition to that consider update to version ^7.21.1 to match main

Reply:
It got a warning from one of the tasks I ran, maybe the storybook - but now I moved it back